sound/hybrid_font.h

```diff
@@ -62,11 +68,15 @@ class HybridFont : public SaberBase {
     lock_player_.Free();
     hum_player_.Free();
     next_hum_player_.Free();
+    swing_player_.Free();
+    second_swing_player_.Free();
     SaberBase::Unlink(this);
   }
 
   RefPtr<BufferedWavPlayer> hum_player_;
   RefPtr<BufferedWavPlayer> next_hum_player_;
+  RefPtr<BufferedWavPlayer> swing_player_;
```

Comment:
maybe just make an array:

RefPtr<BufferedWavPlayer> swing_players_[2];

?

Reply:
decided to have the 'second' swing player not persistent.